src/localization/bundles/en.json

```diff
@@ -393,5 +396,9 @@
     "model.summary.role.description.OVSVAPP-ROLE" : "Describe OVSVAPP Nodes",
     "model.summary.role.description.IRONIC-COMPUTE-ROLE" : "Describe Ironic Compute Nodes",
     "model.summary.role.description.NOT_FOUND" : "Describe the fact that the customer must have created this role",
-    "model.summary.role.component.NOT_FOUND": "Custom Component Nodes"
+    "model.summary.role.component.NOT_FOUND": "Custom Component Nodes",
+
+    "tooltip.network.addresses": "IP addresses used to limit the network access. Addresses could be an IP range or an IP address. For example, 1.2.2.4-1.2.4.5 or 1.2.6.4",
+    "tooltip.network.cidr": "CIDR for the network in the format of 1.2.2.3/4",
```

Comment:
``1.2.2.3/4`` is neither a valid CIDR or good formatting example. It would be better to use text like 'Network address in Classless Inter-Domain Routing notation, such as 10.0.1.0/24'.

Reply:
ok, will fix the example.